docs/results/dot_notation_with_non_ASCII_key.html

```diff
@@ -13,40 +13,173 @@ <h2>Setup</h2>
 
 <h2>Results</h2>
 
-<h3 id="consensus">Consensus</h3>
+<h3>Other responses</h3>
+
+<h4 id="C_json-glib">
+C (json-glib)
+</h4>
 
 <pre><code>[
   "value"
 ]
 </code></pre>
 
-<h3>Other responses</h3>
-
 <h4 id="Clojure_json-path">
 Clojure (json-path)
 </h4>
 
 <pre><code>[
   {
-    "\u5c6c\u6027": "value"
+    "??????": "value"
```

Comment:
This and the following errors indicate a text encoding issue. This might be a problem on your end, e.g. which locale Docker will pick by default.

Could you try re-running by adding the following close to the top of the Dockerfile?
```
ENV LC_ALL en_US.UTF-8
ENV LANG en_US.UTF-8
ENV LANGUAGE en_US.UTF-8
```

You can then trigger a rebuild of that particular query with

    rm -r build/results/dot_notation_with_non_ASCII_key